include/alpaka/vec/Vec.hpp

```diff
@@ -256,7 +256,7 @@ namespace alpaka
             //-----------------------------------------------------------------------------
             auto operator=(Vec &&) -> Vec & = default;
             //-----------------------------------------------------------------------------
-            ~Vec() = default;
+            ALPAKA_FN_HOST_ACC ~Vec() = default;
```

Comment:
do we have a test that uses one of these unlabeled copy/... ctors? The Vec class constructor and destructor have the correct annotation, where the latter one must unify all the used annotations of the ctors. If copy constructor does not have any signature, I think HCC is able to derive it. It would be however a problem, when a copy ctor is flagged as host-only, while it is used within a host-device function, or when the class constructor is device-only. 
I can make it looking more consistent, but it is still not consistent throughout the whole alpaka code. Because in almost all cases I only touched those lines, HCC was complaining about. And I tested the changes back on the HIP branch on Intel and AMD, so I wonder why the ViewPlainPtr passed. 
Well, I will do another iteration on this next week. Would you like to see the consistency on only these classes above, or also on other classes as well, which is not required for HIP(HCC) to work?

Reply:
For now it is enough if those classes you touched are consistent.